### ISSUE TITLE
Reactions are incorrectly positioned on chat bubbles

### ISSUE THREAD

**Issue by andreaslydemann** [GetStream/stream-chat-react-native#2030]
**Describe the bug**
Whenever I add a reaction to a chat bubble that isn't my own message, the reaction is positioned too far to the right. It should be positioned on the edge of the chat bubble, just like when a reaction is added to any of my messages.

**Dev environment info (please complete/provide the following information):**

- stream-chat-expo: 5.7.0
- expo: 46.0.20
- react-native": 0.70.5
- iOS device (iOS 16.2)

**To Reproduce**

Add a reaction to a message from someone else than yourself. Notice how it's not positioned correctly. It's positioned far to the right of the chat bubble. I even tried in the iMessage clone from react-native-samples and I see the same problem there. I attached a screenshot from that project.

**Expected behavior**
I expect the reaction to be positioned the same place as when a reaction is added to my own messages (right on the edge of the chat bubble without any spacing between the reaction and the chat bubble).

![Simulator Screen Shot - iPhone 14 - 2023-03-28 at 19 25 24](https://user-images.githubusercontent.com/15194614/228903609-b14d6b39-d825-4d09-af75-31f9265dc3e3.png)

**Comment by khushal87** [comment]
Hello @andreaslydemann, thanks for reporting this bug. This happens in the iMessagClone because we set `MessageAvatar` to null. I think in your use case, too, you might be setting the MessageAvatar to null, which might be the reason this is happening. Although I agree that this shouldn't happen if we set the message avatar to null since a customer can have this kind of implementation. We will try to fix this is the future.
